src/components/Send.jsx

```diff
@@ -139,22 +118,24 @@ export default function Send({ makerRunning, coinjoinInProcess }) {
 
   const location = useLocation()
   const [alert, setAlert] = useState(null)
+  const [isLoading, setIsLoading] = useState(true)
   const [isSending, setIsSending] = useState(false)
   const [isCoinjoin, setIsCoinjoin] = useState(false)
   const [isCoinjoinOptionEnabled, setIsCoinjoinOptionEnabled] = useState(!makerRunning && !coinjoinInProcess)
+  const [minNumCollaborators, setMinNumCollaborators] = useState(4) // default value from an unchanged joinmarket.cfg (last check 2022-02-20)
 
   const initialDestination = null
   const initialAccount = 0
   const initialAmount = null
-  const initialNumCollaborators = () => {
-    return pseudoRandomNumber(5, 7)
+  const initialNumCollaborators = (minValue) => {
+    return minValue + pseudoRandomNumber(1, 3)
```

Comment:
The way we're shipping the UI, the default value will remain 4 right? In the regtest setup it's 1 (naturally) so the default could be as low as 2. That feels super weird (but doesn't matter in the regtest setup of course). Just wanted to double check that we won't ship a version that recommends a low value such as 2 to the user.

Reply:
The default value is whatever the server repo chooses. We cannot control what a user will subsequently change the value to. In the future we might also let the user change the value in the UI, so we should expect it to have any arbitrary value.

But to answer your question: Yes, we do not change the value in our `standalone` image and hence it will take the current default, which is `4`. But of course the user can always overwrite it by providing a `jm_minimum_makers` env variable.

Reply:
Great idea! Thank you!

Updated! Is this what you had in mind?

Reply:
Yeah I think that's a pretty nice solution now. 👍